python/pyarrow/tests/test_pandas.py

```diff
@@ -2609,7 +2609,8 @@ def test_from_numpy_nested(self):
                        ('y', np.int16),
                        ('z', np.object_)])
         # Note: itemsize is not a multiple of sizeof(object)
```

Comment:
Let's fix this comment
```suggestion
        # Note: itemsize is not necessarily a multiple of sizeof(object)
```